src/services/todo-lists-service.js

```diff
@@ -116,10 +141,51 @@ class TodoListsService {
                     })
             }
             else
-                this.error("Unable to update list because is being modified by another user")
+                Utils.log(`Could not update list ${id} to cluster. Will update local list`);
+
+                var todoListsResponse = nodesService.getAllButSelf().map(node => {
+                    return axios.get(`${Utils.getUrlForPort(node)}/lists/${id}`)
+                })
+
+                Promise.all(todoListsResponse)
+                    .then(todoListsResponse => {
+                        var todoLists = todoListsResponse.map(todoList => {return todoList.data})
+                        var quorumList = this.getQuorumList(todoLists)
+
+                        if (!quorumList)
+                            throw `No quorum achieved for list ${id}!`
+
+                        listRepository.updateToDoList(id, quorumList)
+                    })
+                    .catch(error => {
+                        Utils.log(`Error reading from quorum for list ${id}`, error.response.data)
+                    })
+
+                return {
+                    isOk: false,
+                    message: "Unable to update list because is being modified by another user"
+                }
         })
     }
 
+    requiredQuorum(isRead=false) {
+        const nodes = isRead ? nodesService.getAllButSelf() : nodesService.get()
+
+        return Math.floor(nodes.length / 2)
+    }
+
+    getQuorumList(todoLists) {
+        var groupedToDoLists = Utils.groupBy(todoLists, "hashVersion")
+
+        var listByQuorum = Object.values(groupedToDoLists)
+            .filter(toDoList => {return toDoList.length >= this.requiredQuorum(true) + 1})
```

Comment:
```suggestion
            .filter(toDoList => { return toDoList.length >= this.requiredQuorum(true) })
```
No deberia ser asi? Que pasa si solo hay dos nodos? No podria leer por quorum, nunca lo tendria

Reply:
Este caso límite está manejado en la funcion de requiredQuorum. Si hay 2 nodos, tienen que estar puestos de acuerdo entre ellos. Dejo una tablita para aclarar. Estoy contando los nodos salvo el que se levanta, porque no paraticipa del quorum

1  nodo -> length >= 0 + 1: funciona porque solo requiere la lectura del único nodo levantado
2 nodos -> length >= 1 + 1: funciona porque requiere que los 2 nodos tengan la misma data (sino no se puede tomar una decisión cierta)